packages/trader/src/Stores/Modules/SmartChart/chart-barrier-store.ts

```diff
@@ -94,10 +93,6 @@ export class ChartBarrierStore {
         this.onChartBarrierChange?.(...(barriersToString(this.relative, high, low) as [string, string | undefined]));
     }
 
-    updateBarrierColor(is_dark_mode: boolean) {
```

Comment:
@akmal-deriv according to sonarcloud `BARRIER_COLORS`, you got an unused import in this file

Reply:
Sure, remove it